packages/conveyer/src/Conveyer.ts

```diff
@@ -447,6 +450,11 @@ class Conveyer extends Component<ConveyerEvents> {
           this._checkNestedMove(nativeEvent);
         }
       },
+      "animationEnd": e => {
+        if (!e.isTrusted && this._pos !== this._animationPos) {
```

Comment:
Think about decimal points.

If you do it wrong, it's easy to fall into an infinite loop.

Reply:
"animationEnd": e => {
        if (!e.isTrusted && this._pos !== this._animationPos) {
          this.scrollTo(this._animationPos);
        }
      },
Since this.scrollTo(this._animationPos); triggers this._axes!.setBy with zero duration, There is no chance for the animationEnd event to occur one more time during this process.

https://github.com/naver/egjs-axes/blob/master/packages/axes/src/animation/AnimationManager.ts#L265-L271

If duration passed to setBy is 0, no animation events are fired because the coordinates change immediately without the need to play the animation.

Reply:
In the meantime, I'm working on reducing the coordinate difference while the animation is playing.
I will request a review when I'm finished.